File: RealEstate.Core/Entities/Property.cs
namespace RealEstate.Core.Entities
{
    public class Property: BaseEntity
    {
        public string Name { get; set; }
        public decimal Price { get; set; }
        public float Area { get; set; }
        public int OwnerId { get; set; }        
        public int CategoryId { get; set; }
        public int CityId { get; set; }        
        public string Address1 { get; set; }
        public string Address2 { get; set; }
        public string PostalCode { get; set; }

        public Owner Owner { get; set; }        
        public PropertyCategory Category { get; set; }
        public City City { get; set; }
    }
}

File: RealEstate.UI.AngularWeb/RealEstate/src/app/models/property.ts
import { Address } from "./address";
import { Owner } from "./owner";
import { PropertyCategory } from "./propertyCategory";

export class Property{
    id:number;
    name:string;
    price:number;
    area:number;
    ownerId:number;
    cityId:number;
    addressId:number;
    categoryId:number;
    owner:Owner;
    city:string;
    address1:string;
    address2:string;
    postalCode:string;
    propertyCategory:PropertyCategory;
}